### DISCUSSION TITLE
How to make more than 18 keys?

### DISCUSSION THREAD

**Discussion by Gexant** [Leather128/LeathersFunkinEngine#163] [Q&A]
On the gamebanana page it says "It has many features like 1-18 (but can be extended) keys". But I don't know how to. Help would be much appreciated.

**Comment by sonicfan888** [comment]
#177

**Comment by JadenMudge** [comment]
**_evil_**

**Comment by Farcry5AttackersGames** [comment]
in your mod in the mods folder go to data, mania data there should be 4 text documents it should be binds, animation directions, character animations and mania directions. open and edit all of them. (use the text already put in there as a reference) when your happy with your key binds your not done yet. if you try going in game and see your custom keys they wont appear because your missing something. if you go back to where you open the game go to assets, ui skins, default scroll down or find 3 more documents. mania gap, mania offsets and mania size. add an extra number and make that a slight adjustment. now if you go in game it should appear. if the arrows appear offscreen in a song all you have todo is make some adjustments until the arrows fit on screen.
